src/node_options.cc

```diff
@@ -627,6 +627,11 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "set environment variables from supplied file",
             &EnvironmentOptions::env_file);
   Implies("--env-file", "[has_env_file_string]");
+  AddOption("--override-env-var",
+            "override environment variables set on machine with variables from "
+            "supplied file",
```

Comment:
there needs to be a description with the usage of env-file flag or loadEnvFile

Reply:
@anonrig could you check if it's correct please ?